embulk-input-ftp/src/test/java/org/embulk/input/TestFtpFileInputPlugin.java

```diff
@@ -1,5 +1,314 @@
 package org.embulk.input;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import org.embulk.EmbulkTestRuntime;
+import org.embulk.config.ConfigDiff;
+import org.embulk.config.ConfigSource;
+import org.embulk.config.TaskReport;
+import org.embulk.config.TaskSource;
+import org.embulk.input.FtpFileInputPlugin.PluginTask;
+import org.embulk.spi.Exec;
+import org.embulk.spi.FileInputPlugin;
+import org.embulk.spi.FileInputRunner;
+import org.embulk.spi.InputPlugin;
+import org.embulk.spi.Schema;
+import org.embulk.spi.TestPageBuilderReader;
+import org.embulk.spi.TestPageBuilderReader.MockPageOutput;
+import org.embulk.spi.util.Pages;
+import org.embulk.standards.CsvParserPlugin;
+import org.embulk.util.ftp.SSLPlugins;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeNotNull;
+
 public class TestFtpFileInputPlugin
 {
+    private static String FTP_TEST_HOST;
+    private static Integer FTP_TEST_PORT;
+    private static Integer FTP_TEST_SSL_PORT;
+    private static String FTP_TEST_USER;
+    private static String FTP_TEST_PASSWORD;
+    private static String FTP_TEST_SSL_TRUSTED_CA_CERT_FILE;
+    private static String FTP_TEST_SSL_TRUSTED_CA_CERT_DATA;
+    private static String FTP_TEST_DIRECTORY;
+    private static String FTP_TEST_PATH_PREFIX;
+    private FileInputRunner runner;
+    private TestPageBuilderReader.MockPageOutput output;
+
+    /*
+     * This test case requires environment variables
+     *   FTP_TEST_HOST
+     *   FTP_TEST_USER
+     *   FTP_TEST_PASSWORD
+     *   FTP_TEST_SSL_TRUSTED_CA_CERT_FILE
+     */
+    @BeforeClass
+    public static void initializeConstant()
```

Comment:
`FTP_TEST_SSL_PORT`, `FTP_TEST_SSL_TRUSTED_CA_CERT_FILE` and `FTP_TEST_DIRECTORY` seem not to be used. Are these variables going to be used future?

Reply:
> Are these variables going to be used future?
Yeah and I often use them when I develop at my local env.
But other developers usually don't have FTPS environments.